public/javascripts/SVLabel/src/SVLabel/ribbon/RibbonMenu.js

```diff
@@ -121,10 +121,15 @@ function RibbonMenu(overlayMessageBox, tracker, uiRibbonMenu) {
                     svl.map.modeSwitchLabelClick();
                 }
 
-                // Change cursor before mouse is moved
+                // Change cursor before mouse is moved.
                 if (svl.ui.canvas.drawingLayer) {
                     svl.ui.canvas.drawingLayer.triggerHandler('mousemove');
                 }
+
+                // Load sound effect early.
```

Comment:
Can you add either a description of why we are loading the audio early, or a link to the Github issue that explains it?

Reply:
Is that okay?

Reply:
Yep, looks good!